src/dft/vectortype.h

```diff
@@ -37,11 +37,7 @@
 #include "helpersve.h"
 #endif
 
-#ifdef ENABLE_RVVM1
-#include "helperrvv.h"
-#endif
-
-#ifdef ENABLE_RVVM2
+#if defined(ENABLE_RVVM1) || defined(ENABLE_RVVM1NOFMA) || defined(ENABLE_RVVM2) || defined(ENABLE_RVVM2NOFMA)
```

Comment:
Oh, I see what you mean.  _Should_ it be that way?  Is DFT without FMA an inappropriate configuration?  Like, maybe it won't get past the unit tests or something?

Reply:
I think it would be cleaner that way, as it is not enabled for any other vector extension (in this fail or in CMakeLists.txt). That is because *NOFMA extensions are here for reproducibility in libm routines, there is no such thing as reproducible/deterministic DFT in SLEEF, I believe.

Reply:
OK.  So, by implication I should also remove the NOFMA variants from the libquad work that I'm doing right now, then?

Reply:
Yes, I don't see an occurrence of NOFMA in currently dispatched vector extension for quad, so not needed there either.